app/routes/circulars._archive._index.tsx

```diff
@@ -291,7 +340,152 @@ export default function () {
           </Button>
         </Form>
         {featureCircularsFilterByDate && (
-          <DateSelectorButton startDate={startDate} endDate={endDate} />
+          <>
+            <DateSelectorButton
+              startDate={startDate}
+              endDate={endDate}
+              onClick={() => {
+                setShowContent((shown) => !shown)
+                setShowDateRange(false)
+              }}
+              expanded={showContent}
+            />
+            {showContent && (
+              <DetailsDropdownContent className="margin-top-1">
+                <div className="maxw-card-xlg">
```

Comment:
```suggestion
              <DetailsDropdownContent className="maxw-card-xlg">
                <CardBody>
```

- Remove explicit margin-top; see #1508.
- Enclose content in CardBody.